### PR TITLE
Generalise across sources

### PR DESCRIPTION
**Pull request by seabbs**
Wanted to look across sources so hacked something together. It is pretty crude.

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -21,7 +21,12 @@
 	// Add the IDs of extensions you want installed when the container is created.
 	"extensions": [
 		"ikuyadeu.r",
```
Comment (sbfnk) [inline .devcontainer/devcontainer.json:23]: what are these?

**Review by sbfnk — CHANGES_REQUESTED** [review]
I really like the idea of making these additional plots but I'm not sure adding more plots to the README without explanation is particularly user-friendly. Maybe these could live in separate reports, or this all could be turned into a more structured report?

Also is it an issue that the admissions estimates only get updated once a week, so that the estimates based on them usually refer to different time periods from the others?

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (seabbs) [reply .devcontainer/devcontainer.json:23]: vscode extensions that turn it into an R ide.

**Comment by seabbs** [comment]
yes for sure. I can also just fork elsewhere if easier. 

Agree lack of parity between sources makes comparison quite tricky. Getting around that seems kind of hard beyond a note about when the data is updated.

**Comment by sbfnk** [comment]
Made some tweaks and now merging as I've become convinced of their utility. I don't know how to add the `magrittr` dependency with the `renv` set up so the action will probably fail.

**Pull request merged**